File: modules/stop/src/main/java/com/media/bus/stop/healthcheck/StopServiceHealthCheck.java
package com.media.bus.stop.healthcheck;

import org.springframework.http.ResponseEntity;
import org.springframework.web.bind.annotation.GetMapping;
import org.springframework.web.bind.annotation.RequestMapping;
import org.springframework.web.bind.annotation.RestController;

@RestController
@RequestMapping("/api/v1/stop")
public class StopServiceHealthCheck {

    @GetMapping("/health-check")
    public ResponseEntity<Void> healthCheck() {
        return ResponseEntity.ok().build();
    }


}

File: modules/stop/src/main/kotlin/com/media/bus/stop/healthcheck/StopServiceHealthCheck.kt
package com.media.bus.stop.healthcheck

import org.springframework.http.ResponseEntity
import org.springframework.web.bind.annotation.GetMapping
import org.springframework.web.bind.annotation.RequestMapping
import org.springframework.web.bind.annotation.RestController

@RestController
@RequestMapping("/api/v1/stop")
class StopServiceHealthCheck {

    @GetMapping("/health-check")
    fun healthCheck(): ResponseEntity<Void> = ResponseEntity.ok().build()
}
